monai/transforms/spatial/dictionary.py

```diff
@@ -662,6 +666,7 @@ def __init__(
             shear_params=shear_params,
             translate_params=translate_params,
             scale_params=scale_params,
+            affine=affine,
```

Comment:
Thanks @Nic-Ma . Could you also add unittest for it, and also for `class Affine`?

Reply:
It seems `class AffineGrid` also misses the corresponding unit test for `affine` parameter, could you please add it as well?

Reply:
Sure, I will add it soon.

Thanks for your finding.

Reply:
Added unit tests in the latest commit.

Thanks.